core/src/mindustry/entities/bullet/BulletType.java

```diff
@@ -175,6 +176,8 @@
     public float lightOpacity = 0.3f;
     public Color lightColor = Pal.powerLight;
 
+    public @Nullable Sortf unitSort;
```

Comment:
Shouldn't this just be `Unit::dst2` instead of nullable? You default to it anyway.

Reply:
Well with the code in the Turret, if it were `Unit::dst2` it would override whatever unit sort the turret has.
If it were `Unit::dst2` by default it would override the Foreshadow's unit sort.